src/commands/convert.ts

```diff
@@ -63,9 +70,22 @@ export default class Convert extends Command {
       register.push(...msStatus)
     }
 
+    // adds services
+    servicePath = servicePath ?? await CliUx.ux.prompt(
+      'Services path (optional)',
```

Comment:
```suggestion
      'Please provide the path to the service files source code (optional)',
```

Reply:
suggestion from @jyunmitch

Reply:
@avdev4j, @jyunmitch  
just a note: what do you think if we took away the words "source code"? because docker compose files are actually yaml files (no source)